relay-general/src/protocol/security_report.rs

```diff
@@ -183,7 +183,7 @@ fn normalize_uri(value: &str) -> Cow<'_, str> {
 #[serde(rename_all = "kebab-case")]
 struct CspRaw {
     #[serde(skip_serializing_if = "Option::is_none")]
-    effective_directive: Option<CspDirective>,
+    effective_directive: Option<String>,
```

Comment:
Change it to a string so it won't be implicitly parsed immediatly into CSP-directive, but instead we can handle the cases of the old format.